src/hal/getpinmap_catena4617.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+
+Module:  getconfig_catena4617.cpp
+
+Function:
+        Arduino-LMIC C++ HAL pinmaps for various boards
+
+Copyright & License:
+        See accompanying LICENSE file.
+
+Author:
+        Lakshmi Priya Natarajan, MCCI       June 2019
+
+*/
+
+#if defined(ARDUINO_MCCI_CATENA_4617) || \
+    /* legacy name */ \
+    defined(ARDUINO_CATENA_4617)
```

Comment:
There is no such legacy name.

Reply:
Yes, this legacy name is added by mistake, I will remove it.